cernopendata_client/downloader_class.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# This file is part of cernopendata-client.
```

Comment:
Do we need a new file? The class is relatively small, and if we introduce subclasses, the code will be nicely separated in that way.  So I guess it can stay in the same `downloader.py` file.  It would be simpler and more consistent with the other files present in the cernopendata-client package.

Reply:
I created the new file just to have the class separated to just show how it will look like, will move it to `downloader.py`.